drivers/wifi/esp32/src/esp_wifi_drv.c

```diff
@@ -729,19 +763,19 @@ static int esp32_wifi_dev_init(const struct device *dev)
 }
 
 static const struct wifi_mgmt_ops esp32_wifi_mgmt = {
-	.scan		   = esp32_wifi_scan,
-	.connect	   = esp32_wifi_connect,
-	.disconnect	   = esp32_wifi_disconnect,
-	.ap_enable	   = esp32_wifi_ap_enable,
-	.ap_disable	   = esp32_wifi_ap_disable,
-	.iface_status	   = esp32_wifi_status,
+	.scan          = esp32_wifi_scan,
+	.connect       = esp32_wifi_connect,
+	.disconnect    = esp32_wifi_disconnect,
+	.ap_enable     = esp32_wifi_ap_enable,
+	.ap_disable    = esp32_wifi_ap_disable,
+	.iface_status  = esp32_wifi_status,
 #if defined(CONFIG_NET_STATISTICS_WIFI)
-	.get_stats	   = esp32_wifi_stats,
+	.get_stats     = esp32_wifi_stats,
 #endif
 };
 
 static const struct net_wifi_mgmt_offload esp32_api = {
-	.wifi_iface.iface_api.init	  = esp32_wifi_init,
+	.wifi_iface.iface_api.init = esp32_wifi_init,
```

Comment:
This is just whitespace noise, please remove.

Reply:
Thanks for your feedback about the whitespace noise. I've already removed it from the commit.
By the way, could you let me know when or in what situations I can remove such whitespace?

Reply:
Well, if it was intentional, then you can move that to a separate commit with commit logs detailing the reason. The code review is done per-commit esp. for large PR, so, thumb rule is one logical change per-commit.